### ISSUE TITLE
Player Resize pushes "reccomended" column underneath the page header

### ISSUE THREAD

**Issue by Zarggg** [YePpHa/YouTubeCenter#1761]
![](http://i.imgur.com/2LbLeZk.png)

Screenshot was taken with "static header" and "player gap" off, but neither setting seems to affect this behavior.

Using Firefox 38 beta.

**Comment by YePpHa** [comment]
What is your player size settings?

**Comment by Zarggg** [comment]
I reset all settings to internal defaults before checking this. Settings under Player Size are as follows:
Enable resize: On
Default player size: Default
Small resize button: Default
Large resize button: 720p
Player aspect: 16:9

**Comment by YePpHa** [comment]
Do you get the issue with the experimental player? You can try it out at [https://www.youtube.com/testtube](https://www.youtube.com/testtube).

**Comment by Zarggg** [comment]
No, it seems to be working properly with the experimental player. The issue only occurs with the default player.

**Comment by YePpHa** [comment]
hmm. I tried with the default player and I couldn't recreate your issue.

**Comment by brunobxb** [comment]
I have a similar issue, the "views counter"  is not showing, I've tried to change the position with the UI placement thing but it still does not shows up. Resizing the player still carries the issue.
![816054_8221924 1](https://cloud.githubusercontent.com/assets/12357956/7546260/18f91dc4-f599-11e4-82c2-8535903da86f.png)

**Comment by Zarggg** [comment]
The following sequence always causes the issue to occur:
1. Reset YTC settings to default.
2. Change "Small resize button" setting to "Default"
3. Toggle "Theater/Default mode" button to Default or refresh the page

**Comment by Zarggg** [comment]
Just took a look through Firefox's Inspector to see what might be causing it, and it seems that `watch7-sidebar-contents` needs `margin-top` offset by +30px to line it up directly with the top of the video player.

**Comment by PASAf** [comment]
@brunobxb how did you do that dark theme?

**Comment by brunobxb** [comment]
@PASAf I use this style: https://userstyles.org/styles/92164/youtube-lights-out-a-dark-youtube-theme

**Comment by YePpHa** [comment]
@Zarggg I will see if it can somehow figure out a way to recreate your issue.

**Comment by YePpHa** [comment]
@brunobxb take a look at #1768.

**Comment by brunobxb** [comment]
@YePpHa Thanks, it worked just fine. I edited the css sheet...
